lib/rules/jsx-indent.js

```diff
@@ -39,7 +39,7 @@ module.exports = {
       category: 'Stylistic Issues',
       recommended: false
     },
-
+    fixable: 'whitespace',
```

Comment:
can we add a comment here detailing what cases are not covered? Ideally we'd also add skipped/commented-out test cases for that future work.